modules/server/src/network/__tests__/index.test.ts

```diff
@@ -0,0 +1,131 @@
+// @ts-nocheck
+// graphql types have readonly modifiers all the way down
+
+import { buildSchema } from 'graphql';
+import { createNetworkAggregationTypeDefs } from '../schema';
+
+/**
+ *
+ * @param typeDefs - Array of DefinitionNodes representing the schema type definitions
+ * @param typeName - ObjectTypeDefinition
+ * @returns true if type is defined
+ */
+const isTypeDefined = (typeDefs, typeName) => {
+	return typeDefs.some(
+		(definition) =>
+			definition.kind === 'ObjectTypeDefinition' && definition.name.value === typeName,
+	);
+};
+
+/**
+ *
+ * @param typeDefs - Array of DefinitionNodes representing the schema type definitions
+ * @param parent - ObjectTypeDefinition name
+ * @param child - FieldDefinition name
+ * @returns true if field is defined
+ */
+const isFieldDefined = (typeDefs, parent, child) => {
+	const parentTypeDefFields = typeDefs.find(
+		(definition) => definition.kind === 'ObjectTypeDefinition' && definition.name.value === parent,
+	).fields;
+	return parentTypeDefFields.some(
+		(definitions) => (definitions.kind = 'FieldDefinition' && definitions.name.value === child),
+	);
+};
```

Comment:
Can probably move some test utilities to separate files to organize this code better.